api/DeprecationReportBody.json

```diff
@@ -214,7 +256,14 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": false
+              "version_added": "65",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "dom.reporting.enabled",
+                  "value_to_set": "true"
+                }
+              ]
             },
             "firefox_android": "mirror",
             "ie": {
```

Comment:
Further down is a `toJSON` method. The spec IDL includes this in `DeprecationReportBody` and the parent class  [ReportBody](https://w3c.github.io/reporting/#reportbody) base class too. But this method is NOT declared in FF in `DeprecationReportBody` - would it still be updated here because it is inherited?

Reply:
If Firefox doesn't support it, then keeping it set to `false` seems correct to me.